static/js/mp3.js

```diff
@@ -0,0 +1,22 @@
+function leftPad (str, max) {
+  str = str.toString();
+  return str.length < max ? leftPad("0" + str, max) : str;

```

Comment:
I'm 100% sure, that it is not your code :)
It's a pretty funny recursive implementation. I would make just a loop.